apps/list.c

```diff
@@ -68,6 +167,11 @@ static void list_ciphers(void)
         BIO_printf(bio_out, "  %s", EVP_CIPHER_name(c));
         BIO_printf(bio_out, " @ %s\n",
                    OSSL_PROVIDER_name(EVP_CIPHER_provider(c)));
+        if (verbose) {
+            print_param_types(GET, ALGO, EVP_CIPHER_get_param_types(c));
```

Comment:
I think that this would this be clearer skipping the enum arguments and arrays and just passing a title:
```
print_param_types("retrievable algorithm", EVP_CIPER_get_param_types(c));
print_param_types("retrievable operation", EVP_CIPHER_CTX_get_param_types(c));
print_param_types("settable operation", EVP_CIPHER_CTX_set_param_types(c));
```

Reply:
Good point

Reply:
The same sequence appears again later in the file, so it needs to be hit in two places.

Reply:
Done

Reply:
Awaiting push?

Reply:
Oh...  done